detox/test/e2e/03.actions-scroll.test.js

```diff
@@ -1,12 +1,10 @@
-const custom = require('./utils/custom-it');
-
 describe('Actions - Scroll', () => {
   beforeEach(async () => {
     await device.reloadReactNative();
     await element(by.text('Actions')).tap();
   });
 
-  custom.it.withFailureIf.android('should scroll for a small amount in direction', async () => {
+  it('should scroll for a small amount in direction', async () => {
```

Comment:
FINALLY. That's amazing!!

Reply:
I guess we can remove `withFailureIf` entirely now (`withFailureIf.android` is no longer used and the other usage is for RN<67 - we don't test it anymore). but not critical, you can ignore this comment if you want

Reply:
I will keep it. Who knows if we need it in the next Android versions